src/Hl7.Fhir.Support.Tests/Specification/MimeTypeTerminologyServiceTests.cs

```diff
@@ -52,5 +54,51 @@ public async Task MimeTypeValidationTest()
             validateCode = async () => await _service.ValueSetValidateCode(parameters);
             await validateCode.Should().ThrowAsync<FhirOperationException>().WithMessage("If a code is provided, a system or a context must be provided");
         }
+
+        [TestMethod]
+        public async Task MimeTypeValidationWithVersionedVSTest()
+        {
+            var parameters = new ValidateCodeParameters()
+                   .WithValueSet(MIMETYPE_VERSIONED_VS)
+                   .WithCode(code: "invalid", context: "context")
+                   .Build();
+
+            var result = await _service.ValueSetValidateCode(parameters);
+            result.Parameter.Should().Contain(p => p.Name == "message")
+                .Subject.Value.Should().BeEquivalentTo(new FhirString($"'invalid' is not a valid MIME type."));
+
+
+            parameters = new ValidateCodeParameters()
+                   .WithValueSet(MIMETYPE_VERSIONED_VS)
+                   .WithCode(code: "application/json", context: "context")
+                   .Build();
+
+            result = await _service.ValueSetValidateCode(parameters);
+            result.Parameter.Should().Contain(p => p.Name == "result")
+                .Subject.Value.Should().BeEquivalentTo(new FhirBoolean(true));
+        }
+
+        [TestMethod]
+        public async Task MimeTypeValidationWithStu3VS()
```

Comment:
Almost exactly the same code as `MimeTypeValidationWithVersionedVSTest`. Can you make this a unit test with `DataRow`s? The variable is here the ValueSet.